meta-resin-common/recipes-support/resin-mounts/resin-mounts-flasher/temp-conf.service

```diff
@@ -1,13 +1,14 @@
 [Unit]
 Description=Temporary resin config in /tmp/conf
-Requires=mnt-conforig.mount tmp.mount
-After=mnt-conforig.mount tmp.mount
+Requires=mnt-boot.mount tmp.mount
+After=mnt-boot.mount tmp.mount
+Before=resin-device-uuid.service
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 ExecStartPre=@BASE_BINDIR@/mkdir -p /tmp/conf
-ExecStart=@BASE_BINDIR@/cp -vf /mnt/conforig/config.json /tmp/conf/config.json
+ExecStart=@BASE_BINDIR@/cp -vf /mnt/boot/config.json /tmp/conf/config.json || @BASE_BINDIR@/touch /tmp/conf/config.json
```

Comment:
Why do you touch the file in case there isn't one in /mnt/boot/config.json or the cp fails for some reason? What good will an empty config.json do?